src/Microsoft.Data.Sqlite/Interop/NativeMethods.cs

```diff
@@ -26,12 +26,6 @@ internal static class NativeMethods
 #if NET45 || DNX451 || DNXCORE50
         static NativeMethods()
         {
-#if DNXCORE50
-            if(!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
-            {
-                return;
-            }
-#endif
```

Comment:
I still think we should remove it as the `"System.Runtime.InteropServices.RuntimeInformation"` package is fickle. As of yesterday, our CI feed publishes a Windows version. At other times, our CI have contained the Linux version. I don't want CI to break this code.

This may change if we unpublish System.Runtime.InteropServices.RuntimeInformation altogether because I believe this publication is actually a mistake in our CI settings.